cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorPodSetTest.java

```diff
@@ -73,9 +73,11 @@
 import java.util.function.Function;
 import java.util.stream.IntStream;
 
+import static io.strimzi.api.ResourceAnnotations.ANNO_STRIMZI_IO_SKIP_BROKER_SCALEDOWN_CHECK;
```

Comment:
Same as above. Can you use `Annotations.NNO_STRIMZI_IO_SKIP_BROKER_SCALEDOWN_CHECK`?